examples/protocol_specification_ex/sample.yaml

```diff
@@ -7,28 +7,49 @@ license: Apache-2.0
 aea_version: '>=0.3.0, <0.4.0'
 speech_acts:
   cfp:
-    query: pt:str
+    query: ct:Query
   propose:
-    number: pt:int
-    price: pt:float
-    description: pt:str
-    flag: pt:bool
-    query: pt:str
-    proposal: pt:optional[pt:dict[pt:str, pt:str]]
-    rounds: pt:set[pt:int]
-    items: pt:list[pt:str]
-    conditions: pt:optional[pt:union[pt:str, pt:dict[pt:str,pt:int], pt:set[pt:str], pt:dict[pt:str, pt:float]]]
-  request:
-    method: pt:str
-    url: pt:str
-    version: pt:str
-    headers: pt:str
-    bodyy: pt:optional[pt:bytes]
+    proposal: ct:Description
   accept: {}
-  inform:
-    inform_number: pt:list[pt:int]
-  inform_reply:
-    reply_message: pt:dict[pt:int, pt:str]
   decline: {}
```

Comment:
Please don't change this each time, just append it so we can see what's changed with the new PR...

Reply:
It's not really even needed. I can remove it from the repo. 
I use it locally to test new generator functionality.

Reply:
Please keep it and revert the changes above. It is good to have as a demo. Changes below can obviously stay.

Reply:
Done